libs/types/src/election.ts

```diff
@@ -308,15 +308,26 @@ export const PrecinctsSchema = z
     }
   });
 
-export type BallotStyleId = Id;
-export const BallotStyleIdSchema: z.ZodSchema<BallotStyleId> = IdSchema;
+export type BallotStyleId = NewType<string, 'BallotStyleId'>;
+export const BallotStyleIdSchema =
+  IdSchema as unknown as z.ZodSchema<BallotStyleId>;
 export interface BallotStyle {
   readonly id: BallotStyleId;
   readonly precincts: readonly PrecinctId[];
   readonly districts: readonly DistrictId[];
   readonly partyId?: PartyId;
   readonly languages?: readonly LanguageCode[]; // TODO(kofi): Make required.
 }
+
+export type BallotStyleGroupId = NewType<string, 'BallotStyleGroupId'>;
+export interface ParentBallotStyle {
+  readonly id: BallotStyleGroupId;
+  readonly precincts: readonly PrecinctId[];
+  readonly districts: readonly DistrictId[];
+  readonly partyId?: PartyId;
+  readonly languages?: readonly LanguageCode[]; // TODO(kofi): Make required.
+}
```

Comment:
I notice that in some places we use default language ballot styles and in some places we use `ParentBallotStyle`. I think having a mix of both concepts makes things a bit confusing. Can we pick one?

Also, if we go with parent ballot styles, maybe we could call it `BallotStyleGroup` instead and remove the languages field (keeping only the fields that are common across all of the ballot styles in the group).